modules/utxo-lib/src/bitgo/Unspent.ts

```diff
@@ -124,6 +124,61 @@ export function addToTransactionBuilder<TNumber extends number | bigint>(
   txb.addInput(txid, vout, sequence, script, value);
 }
 
+export function addChangeOutputToPsbt(
+  psbt: UtxoPsbt<UtxoTransaction<bigint>>,
+  rootWalletKeys: RootWalletKeys,
+  chain: ChainCode,
+  index: number,
+  value: bigint
+): void {
+  const walletKeys = rootWalletKeys.deriveForChainAndIndex(chain, index);
```

Comment:
We're allowing both internal and external chain codes here iiuc. Do we want to require the chaincode be internal, as I would expect for change outputs? Or alternatively we can accept a `scriptType` and then do `getInternalChainCode()` within this function.

Reply:
Good call.

Reply:
Nix that. The chain is an API parameter in WP, so we need to accept chain on the addChangeOutput. I'll add a comment.

Reply:
Gotcha, maybe from WP we could have a check that the chain is internal?